helm/README.md

```diff
@@ -186,48 +186,47 @@ information.
-| serviceMonitor.namespace                     | The namespace Prometheus servicemonitor will be installed                                             | str      | `.Release.Namespace`                                       |
-| serviceMonitor.additionalLabels              | Additional labels for the servicemonitor                                                              | map      | `-`                                                        |
-| serviceMonitor.relabelings                   | Additional metrics relabel_config                                                                     | lists    | `-`                                                        |
-| nameOverride                                 | Override default chartname                                                                            | str      | `""`                                                       |
-| fullnameOverride                             | Override default releasename                                                                          | str      | `""`                                                       |
+| Parameter                               | Description                                                                                           | Type     | Default                                                    |
+| ---                                     | ---                                                                                                   | ---      | ---                                                        |
+| nameOverride                            | Override default chartname                                                                            | str      | `""`                                                       |
+| fullnameOverride                        | Override default releasename                                                                          | str      | `""`                                                       |
+| rbac.create                             | if `true`, create and use RBAC resources                                                              | bool     | `true`                                                     |
+| serviceAccount.create                   | if `true`, create serviceaccount in .Release.Namespace                                                | bool     | `true`                                                     |
+| serviceAccount.name                     | if set serviceaccount if the given name will be created                                               | str      | `""`                                                       |
+| useAlphaAPI                             | If running against pre-1.10 k8s version, the `useAlphaAPI` flag must be enabled.                      | bool     | `false`                                                    |
```

Comment:
Maybe since we're going to bump a major version, we can remove this as well.

Reply:
I removed the key from the helm chart too, pending is the removal of the go code that uses this field.